README.md

```diff
@@ -157,6 +157,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [QuantumTomography.jl](https://github.com/BBN-Q/QuantumTomography.jl) - Julia package to perform quantum state and process tomography.
 
 **Python**
+- [Arline Quantum](https://github.com/ArlineQ/arline_quantum) - Library with implementation of quantum gates and hardware, a part of [Arline Benchmarks](https://github.com/ArlineQ/arline_benchmarks) project.
```

Comment:
could you please move this below `Adapt` in order to adhere to alphabetical sorting?

Reply:
Thanks for the comment. It was my bad. 
Done, please check.